tendermint/src/block/height.rs

```diff
@@ -7,25 +7,27 @@ use std::{
 
 /// Block height for a particular chain (i.e. number of blocks created since
 /// the chain began)
+/// 
+/// A height of 0 represents a chain which has not yet produced a block.
 #[derive(Copy, Clone, Eq, Hash, PartialEq, PartialOrd, Ord)]
 pub struct Height(u64);
 
 impl Height {
     /// Convert `u64` to block height.
-    ///
-    /// Note that 0 is not a valid block height.
+    /// 
+    /// Note that this method will never error and is just for backwards
+    /// compatibility.
```

Comment:
I can try to address it in a followup PR, but it might be good to just break compatibility here, make the above struct `pub struct Height(pub u64);` for convenience, and add `From` impls between `Height` and `u64`